advanced_functionality/distributed_tensorflow_mask_rcnn/mask-rcnn-s3.ipynb

```diff
@@ -0,0 +1,566 @@
+{
```

Comment:
<p>As mentioned in an earlier comment, could help to set the <code>sagemaker_session</code> in an earlier cell, in order to use the <code>default_bucket()</code> method.</p><p> </p><p>Also nit, could there be a better name for the Estimator variable than <code>obj_detection</code>? Maybe <code>maskrcnn_estimator</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/891/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
As noted above, not using default_bucket method. Will not make this change in the interest of avoiding new round of testing.